observer/api.go

```diff
@@ -123,6 +123,24 @@ func (a *API) GetApprovedHermesAdresses() (map[int64][]common.Address, error) {
 	return res, nil
 }
 
+// GetHermesAdresses returns a map by chain of hermes addresses filtered by the given query.
+func (a *API) GetHermesAdresses(f *HermesFilter) (map[int64][]common.Address, error) {
+	resp, err := a.GetHermeses(f)
+
+	if err != nil {
+		return nil, err
+	}
+	res := make(map[int64][]common.Address)
```

Comment:
This is the same loop as in `GetApprovedHermesAdresses` and this filter is very inconvenient to use for a caller because of the pointers inside.

Reply:
now its for sure better but not sure about the filter, for some functions its nice that it does inside, but now it would be `GetApprovedAndActiveHermesAdresses`, it starts to look a bit funny